packages/datamash.rb

```diff
@@ -3,21 +3,13 @@
 class Datamash < Package
   description 'GNU Datamash is a command-line program which performs basic numeric,textual and statistical operations on input textual data files.'
   homepage 'http://savannah.gnu.org/projects/datamash'
-  version '1.1.1'
-  source_url 'https://ftpmirror.gnu.org/datamash/datamash-1.1.1.tar.gz'
-  source_sha256 '420819b3d7372ee3ce704add847cff7d08c4f8176c1d48735d4a632410bb801b'
+  version '1.2'
+  source_url 'https://mirror.sergal.org/gnu/datamash/datamash-1.2.tar.gz'
```

Comment:
Probably should stick with https://ftpmirror.gnu.org/datamash/datamash-1.2.tar.gz since it optimizes based on locality.

Reply:
Just pulled the URL from #1585. I can update.